README.md

```diff
@@ -133,6 +133,93 @@ Additional options:
 * `database`: Initial database to select. Defaults to SQL Server user profile settings.
 * `ssl`: Whether to use transport-level encryption for the entire SQL server traffic, defaults to `false`.
 
+### Data Type Mapping 
+
+[Microsoft Sql Server][m] type mapping reference. This reference table shows encoding/decoding types from [Microsoft Sql Server][m] types to java data types.
+
+
+| Microsoft SQL Server Type               | Supported Data Type                                                                                                              | 
+|:----------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------|
+| [bit][sql-bit-ref]                      | [Boolean*][java-boolean-ref], [Byte][java-byte-ref], [Short][java-short-ref], [Integer][java-integer-ref], [Long][java-long-ref] |
+| [tinyint][sql-all-int-ref]              | [Boolean][java-boolean-ref], [Byte*][java-byte-ref], [Short][java-short-ref], [Integer][java-integer-ref], [Long][java-long-ref] |
+| [smallint][sql-all-int-ref]             | [Boolean][java-boolean-ref], [Byte][java-byte-ref], [Short*][java-short-ref], [Integer][java-integer-ref], [Long][java-long-ref] |
+| [int][sql-all-int-ref]                  | [Boolean][java-boolean-ref], [Byte][java-byte-ref], [Short][java-short-ref], [Integer*][java-integer-ref], [Long][java-long-ref] |
+| [bigint][sql-all-int-ref]               | [Boolean][java-boolean-ref], [Byte][java-byte-ref], [Short][java-short-ref], [Integer][java-integer-ref], [Long*][java-long-ref] |
+| [float][sql-float-real-ref]             | [Double*][java-double-ref], [Float][java-float-ref] 
+| [real][sql-float-real-ref]              | [Double][java-double-ref], [Float*][java-float-ref]   
```

Comment:
We should probably swap `float` and `real` to keep sorting by ascending data type size.